server/utils/utils.ts

```diff
@@ -18,4 +22,44 @@ const convertToTitleCase = (sentence: string): string =>
 
 const hasRole = (user: Express.User, role: AuthRole): boolean => user?.userRoles.includes(role) || false
 
-export { convertToTitleCase, hasRole }
+/**
+ * Converts a SimpleDateTime display value to a JSON string format dd/mm/yyyy hh:mm
+ * @param dt: SimpleDateTime
+ */
+const simpleDateTimeToJson = (dt: SimpleDateTime): string | undefined => {
+  const { inductionDate, inductionTime } = dt
+  const momentDt = moment(
+    [inductionDate.year, inductionDate.month, inductionDate.day].join('-'),
+    ['YYYY-M-D', 'YY-M-D'],
```

Comment:
Why 2 formats?

Reply:
I copied this from the existing code for validating - assumed 2-digit years are also possible on input?

Reply:
Oh yes you are right, sorry. Might need added in again? 😢

Reply:
Unit test added for a 2-digit year